app/views/denounces/_form.html.erb

```diff
@@ -41,12 +41,27 @@
         <div class="col-lg-5">
           <%= form_field.text_field :postal_code, label: "Postal Code",  id: "postalCodeInput"%>
         </div>
-        <div class="col-lg-10">
+        <div class="col-lg-10 overflow-scroll" style="height: 260px">
```

Comment:
Inline styles should be avoided if possible.

Reply:
Thanks, we used it for testing and finally forgot to remove it , but we will fix in general refactor